tests/integration_tests/performance/test_balloon.py

```diff
@@ -147,3 +169,83 @@ def test_hinting_fault_latency(
 
         if sleep_duration > 0 and (i + 1 < runs):
             time.sleep(sleep_duration)
+
+
+# pylint: disable=C0103
```

Comment:
nit: moving the test before making changes can got to a separate commit.

Reply:
Will do thanks!